src/time-picker/time-picker-dialog.jsx

```diff
@@ -70,6 +70,7 @@ const TimePickerDialog = React.createClass({
       root: {
         fontSize: 14,
         color: this.getTheme().clockColor,
+        userSelect: 'none; -webkit-user-select: none; -moz-user-select: none; -khtml-user-select: none; -ms-user-select: none',
```

Comment:
`userSelect: 'none',` should be enough, the autoprefixer will do the job.
Could you move this to the `Clock` component?